CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ CHANGELOG
   * `PAYSAFECARD`
   * `SMARTDEBIT`
   * `SYNAPSEFI`
+* Deprecated `email_tenure` and `ip_tenure` subscores, in addition to `is_high_risk`.
```

Comment:
Use the getter method names. Also, `is_high_risk` applies to more than one class. We are only deprecating the IP address country one.